kyuubi-common/src/test/scala/org/apache/kyuubi/service/TFrontendServiceSuite.scala

```diff
@@ -557,9 +557,7 @@ class TFrontendServiceSuite extends KyuubiFunSuite {
       assert(cancelOpResp.getStatus.getStatusCode === TStatusCode.SUCCESS_STATUS)
       assert(sessionManager.getOpenSessionCount === 1)
       assert(session.lastIdleTime === 0)
-      eventually(timeout(Span(60, Seconds)), interval(Span(1, Seconds))) {
```

Comment:
If the execution speed is very fast, the access time may be equal. This waiting may cause the operation and session to be cleared in advance, and subsequent asserts will fail.

https://github.com/apache/kyuubi/blob/fbe80ace11e4049e5a00737a208dfe6f50e3d370/kyuubi-common/src/main/scala/org/apache/kyuubi/session/SessionManager.scala#L311-L318